tests/unit/test_workflows.py

```diff
@@ -7,3 +13,40 @@ def test_tasks_detected():
     tasks = workflows.tasks()
 
     assert len(tasks) > 1
+
+
+@dataclasses.dataclass
```

Comment:
these look irrelevant to this PR

Reply:
it was with FileProblem, but no longer is